packaging/ubuntu-16.04/snapd.postrm

```diff
@@ -33,6 +33,12 @@ if [ "$1" = "purge" ]; then
         fi
     done
 
+    # Stop user units which are loaded
+    user_units=$(systemctl list-units --no-legend --user "snap.*" | cut -f1 -d ' ' || true)
```

Comment:
A very silly question - does this work? The postinst runs as root and does not have access to the users session bus. Even if it does, it does not have access to all the users session buses. So I wonder if we can have this change ?